pkgs/by-name/nu/nuv/tests.nix

```
@@ -0,0 +1,16 @@
+{ runCommand, nuv, pname, version }:
+
+runCommand "nuv-test-run"
+{
+  nativeBuildInputs = [ nuv ];
+} ''
+  export HOME=/tmp
+  export NUV_REPO=""
+  export NUV_ROOT=/tmp/.nuv/3.0.0/olaris
```

can you use mktemp https://www.gnu.org/software/autogen/mktemp.html?

```suggestion
  export NUV_ROOT=$(mktemp -d /tmp/.nuv-XXXXX)
```